resources/setup.sh

```diff
@@ -99,8 +106,8 @@ kubectl -n kube-system create -f coredns-cm.yaml
 rm -f coredns-cm.yaml
 
 # expose kube config so external consumers can call in
-cp /root/.minikube/client.crt /var/kube-config/client.crt
-cp /root/.minikube/client.key /var/kube-config/client.key
+cp /root/.minikube/proxy-client-ca.crt /var/kube-config/client.crt
```

Comment:
does this need a condition / guard as well? is it `proxy-client..` for all versions?

Reply:
To be frank, I am not sure if these are the right certificates. My feeling is to remove that completely as the needed information is in /root/.kube/config

Reply:
ah yes it is too, has been for ages too, should be safe to drop it :+1: